android/src/main/java/com/swmansion/gesturehandler/core/PanGestureHandler.kt

```diff
@@ -212,6 +213,14 @@ class PanGestureHandler(context: Context?) : GestureHandler<PanGestureHandler>()
       return
     }
 
+    if (state == STATE_UNDETERMINED) {
+      stylusData.pressure = -1.0
+    }
```

Comment:
Shouldn't this be inside `onReset`? And please make a new object.

Reply:
Done in [b896f2e](https://github.com/software-mansion/react-native-gesture-handler/pull/3111/commits/b896f2edc7161176ecad6b9d7fab165229be139b)